tests/jeos/firstrun.pm

```diff
@@ -67,8 +67,16 @@ sub verify_mounts {
 
 sub run {
     my ($self) = @_;
-
     mouse_hide;    # JeOS on generalhw
+                   # kiwi-templates-JeOS images (sle, opensuse x86_64 only) are build w/o translations
+                   # jeos-firstboot >= 0.0+git20200827.e920a15 locale warning dialog has been removed
```

Comment:
Indentation looks wrong. You probably need to add a blank line after `mouse_hide` to be able to have a proper indentation of the comments.

Reply:
yeah, that is how my tidy script aligns comments. I have moved the generalhw comment above the call.